assets/panel/panel.js

```diff
@@ -16,49 +15,45 @@ function DebuggerPanel(iframeWindow, toolbox) {
 }
 
 DebuggerPanel.prototype = {
-  open: Task.async(function* () {
+  open: Task.async(function*() {
```

Comment:
I believe it should be safe to do so here... Firefox supports async / await natively now.